index.js

```diff
@@ -928,7 +916,7 @@ function buildObject (location, code, name) {
   if (schema.nullable) {
     code += `
       if(input === null) {
-        return '${$asNull()}';
+        return 'null';
```

Comment:
Why this change?

Reply:
The $asNull function is now inside a string constant, so it is not available to be called in this context.

Reply:
Is it not the case that an alternative function can be provided for `$asNull` and that such an alternative could return a value other than `'null'`?

Reply:
Not to my knowledge? If so I don't see how such a function would be provided.

Reply:
I could change it to `return $asNull()` I guess, but then there are other places in the code which just return `'null'` anyway, so it would still make much difference in terms of consistency.

Reply:
there is still one `$asNull` call. I would remove it and remove the function aswell